tensorflow-framework/src/test/java/org/tensorflow/framework/constraints/UnitNormTest.java

```diff
@@ -0,0 +1,66 @@
+package org.tensorflow.framework.constraints;
+
+import org.junit.jupiter.api.Test;
+import org.tensorflow.Operand;
+import org.tensorflow.framework.utils.TestSession;
+import org.tensorflow.ndarray.Shape;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TFloat64;
+
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicInteger;
+
+class UnitNormTest {
+  private final TestSession.Mode[] tfModes = {TestSession.Mode.EAGER, TestSession.Mode.GRAPH};
+
+  private float[] getSampleArray() {
+    Random rand = new Random(3537L);
+    float[] result = new float[100 * 100];
+    for (int i = 0; i < result.length; i++) {
+      result[i] = rand.nextFloat() * 100 - 50;
+    }
+    result[0] = 0;
+    return result;
+  }
+
+  /** Test of call method, of class MaxNorm. */
```

Comment:
`MaxNorm` -> `UnitNorm` (not just in the comment, but in the code)

Reply:
My mistake, I'll fix.

Reply:
👍 done